### ISSUE TITLE
Build fails at `bundle.parseTranslations`

### ISSUE THREAD

**Issue by MattiSG** [openfisca/openfisca.org#99]
In #96 and #97, [build](https://app.circleci.com/pipelines/github/openfisca/openfisca.org/262/workflows/7d5fafe6-b645-4a88-9ad7-59fafe4bc403/jobs/1154) [fails](https://app.circleci.com/pipelines/github/openfisca/openfisca.org/261/workflows/d015fe8f-e295-4e2c-8771-c77dccc2cfc9/jobs/1152) with the following reason:

```
goroutine 54 [running]:
github.com/gohugoio/hugo/vendor/github.com/nicksnyder/go-i18n/i18n/bundle.parseTranslations(0xc42027aab3, 0x7, 0xc420220140, 0x138, 0x138, 0x429934, 0xc420140600, 0xc42061d7d8, 0xc42061d7c8, 0xc42061d7c8)
	/go/src/github.com/gohugoio/hugo/vendor/github.com/nicksnyder/go-i18n/i18n/bundle/bundle.go:100 +0x7a6
github.com/gohugoio/hugo/vendor/github.com/nicksnyder/go-i18n/i18n/bundle.(*Bundle).ParseTranslationFileBytes(0xc42021e3c0, 0xc42027aab3, 0x7, 0xc420220140, 0x138, 0x138, 0xc42027aaf3, 0x2)
	/go/src/github.com/gohugoio/hugo/vendor/github.com/nicksnyder/go-i18n/i18n/bundle/bundle.go:75 +0xdb
github.com/gohugoio/hugo/i18n.addTranslationFile(0xc42021e3c0, 0xfdbb20, 0xc4201c62a0, 0x0, 0x0)
	/go/src/github.com/gohugoio/hugo/i18n/translationProvider.go:98 +0x25b
github.com/gohugoio/hugo/i18n.(*TranslationProvider).Update(0xc420548d00, 0xc4200f48f0, 0xc4200f48f0, 0xc4200d2f60)
	/go/src/github.com/gohugoio/hugo/i18n/translationProvider.go:78 +0x5d5
github.com/gohugoio/hugo/deps.(*Deps).LoadResources(0xc4200f48f0, 0xc420509dc0, 0xc4200d2960)
	/go/src/github.com/gohugoio/hugo/deps/deps.go:73 +0x43
github.com/gohugoio/hugo/hugolib.applyDepsIfNeeded(0xc42050f400, 0xc420509dc0, 0xc4200d2960, 0xfceba0, 0xc4202a9590, 0xfbe120, 0x0, 0xc42021e0f0, 0xfbe0a0, 0xc420548d00, ...)
	/go/src/github.com/gohugoio/hugo/hugolib/hugo_sites.go:184 +0x388
github.com/gohugoio/hugo/hugolib.newHugoSites(0xc42050f400, 0xc420509dc0, 0x0, 0xfceba0, 0xc4202a9590, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/gohugoio/hugo/hugolib/hugo_sites.go:143 +0x285
github.com/gohugoio/hugo/hugolib.NewHugoSites(0xc42050f400, 0xc420509dc0, 0x0, 0xfceba0, 0xc4202a9590, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/gohugoio/hugo/hugolib/hugo_sites.go:212 +0xf5
github.com/gohugoio/hugo/commands.(*commandeer).initSites(0xc4205a7ab0, 0x0, 0x0)
	/go/src/github.com/gohugoio/hugo/commands/hugo.go:947 +0x66
github.com/gohugoio/hugo/commands.(*commandeer).buildSites(0xc4205a7ab0, 0x0, 0x0)
	/go/src/github.com/gohugoio/hugo/commands/hugo.go:958 +0x2f
github.com/gohugoio/hugo/commands.(*commandeer).fullBuild.func3(0x8, 0xea9150)
	/go/src/github.com/gohugoio/hugo/commands/hugo.go:563 +0x2e
github.com/gohugoio/hugo/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1(0xc420509fc0, 0xc420504ac0)
	/go/src/github.com/gohugoio/hugo/vendor/golang.org/x/sync/errgroup/errgroup.go:58 +0x57
created by github.com/gohugoio/hugo/vendor/golang.org/x/sync/errgroup.(*Group).Go
	/go/src/github.com/gohugoio/hugo/vendor/golang.org/x/sync/errgroup/errgroup.go:55 +0x66

Exited with code exit status 2
```

**Comment by MattiSG** [comment]
This also prevented #95 from being deployed.

**Comment by MattiSG** [comment]
This did not however prevent #95 from being merged. After investigation, it turns out that branch protection rules did not have mandatory checks. I have now switched on mandatory checks for `build` and `test`. I have not switched on mandatory checks for Netlify previews, as the name of these checks is likely to change with #98, since the name of the image is provided in the check name.

**Comment by MattiSG** [comment]
This issue being related to `bundle.parseTranslations`, it might actually have been introduced in #95.

**Comment by MattiSG** [comment]
Fixed by #100, as evidenced by the availability of https://openfisca.org/en/packages/, which was https://openfisca.org/en/countries/ before #95 💪
